gamedata/lua/lua/LEMLobbyOptions.lua

```diff
@@ -41,104 +41,104 @@ GPGrestrictedUnits = {
+		name = "<LOC restricted_units_data_0026>No Air Experimentals",
 		tooltip = "restricted_units_air_experimentals",
     },
 	
 	TACTICALMISSILELAUNCHERS = {
-		categories = { 'uab2108', 'ueb2108', 'urb2108', 'xsb2108', 'bab2308', },
-		name = "No Tactical Missile structures",
-		tooltip = "restricted_units_tactical",
+		categories = {"bab2308", "uab2108", "ueb2108", "urb2108", "xsb2108"},
+		name = "<LOC restricted_units_data_0027>No TMLs",
+		tooltip = "restricted_units_tml",
 	},
 	
     NUKE = {
-        categories = {"uab2305", "ueb2305", "urb2305", "xsb2305", "xsb2401", "uas0304", "urs0304", "uab4302", "ueb4302", "urb4302", "xsb4302", "sal0321", "sel0321", "srl0321", "ssl0321" },
-        name = "<LOC restricted_units_data_0011>No Nukes or Anti-nukes",
+        categories = {"sal0321", "sel0321", "srl0321", "ssl0321", "uab2305", "uab4302", "ueb2305", "ueb4302", "urb2305", "urb4302", "xsb2305", "xsb2401", "xsb4302", "uas0304", "urs0304"},
+        name = "<LOC restricted_units_data_0011>No Nukes",
         tooltip = "restricted_units_nukes",
     },
 	
 	TACTICALARTILLERY = {
-		categories = { "lab2320", "leb2320", "lrb2320", "lsb2320" },
-		name = "No T3 Barrage Artillery",
-		tooltip = "restricted_units_T3_Tactical_artillery",
+		categories = {"lab2320", "leb2320", "lrb2320", "lsb2320"},
+		name = "<LOC restricted_units_data_0028>No T3 Barrage Artillery",
+		tooltip = "restricted_units_T3_tactical_artillery",
 	},
 
     STRATEGICARTILLERY = {
-        categories = { "uab2302","ueb2302","urb2302","xsb2302" },
-        name = "No T3 Strategic Artillery",
-        tooltip = "restricted_units_T3_Strategic_artillery",
+        categories = {"uab2302", "ueb2302", "urb2302", "xsb2302"},
+        name = "<LOC restricted_units_data_0029>No T3 Strategic Artillery",
+        tooltip = "restricted_units_T3_strategic_artillery",
     },
 	
 	EXPERIMENTALARTILLERY = {
-		categories = { "xab2307", "ueb2401", "url0401", "seb2404", "ssb2404" },
-		name = "No Experimental Artillery",
+		categories = {"seb2404", "ssb2404", "ueb2401", "url0401", "xab2307"},
+		name = "<LOC restricted_units_data_0030>No Experimental/T4 Artillery",
 		tooltip = "restricted_units_exp_artillery",
 	},
 
     SHIELDS = {
-        categories = {"uel0307", "ual0307", "xsl0307", "xes0205", "ueb4202", "urb4202", "uab4202", "xsb4202", "ueb4301", "uab4301", "xsb4301","seb4401","ssb4401","ualx0401","uabssg01","uebssg01","urbssg01","xsbssg01" },
-        name = "<LOC restricted_units_data_0013>No Shield Units",
-        tooltip = "restricted_units_bubbles",
+        categories = {"sab4401", "seb4401", "srb4401", "ssb4401", "uab4202", "uab4301", "uabssg01", "ual0307", "ual0308", "ualx401", "ueb4202", "ueb4301", "uebssg01", "uel0307", "uel0308", "urb4202", "urb4207", "urbssg01", "wrl0207", "xes0205", "xsb4202", "xsb4301", "xsbssg01", "xsl0307"},
+        name = "<LOC restricted_units_data_0013>No Shields",
```

Comment:
Referring explicitly to "shields" improves greatly on references to just "bubbles", although the new ambiguity here is in implying that this restriction applies to all shields, personal or bubble. Either "No Bubble Shields" or "No Dome Shields" should be the universally-used wording.

Reply:
"No Shields" is simply the title of the restriction. The description of the restriction which the player will see ties back to tooltips.lua (new line 2107): "Players will not be able to build mobile or stationary shield generators"